lib/archethic/mining/standalone_workflow.ex

```diff
@@ -199,7 +203,7 @@ defmodule Archethic.Mining.StandaloneWorkflow do
       transaction_address: Base.encode16(tx_address)
     )
 
-    Logger.debug("Notify error back to the welcome node",
+    Logger.error("Notify error back to the welcome node",
```

Comment:
This log should stay as debug I think, it is only an information that the node is notifying the welcome node, that is not an error. The error log while validating the transaction is already done before

Reply:
done 👍